### PR TITLE
Add rollup bundler

### PR DESCRIPTION
**Pull request by bddjong**
Have to rewrite the description

### CODE REVIEW

**Comment by mjcarsjens** [comment]
Hmm, looks like a solid first setup but there seem to still be some quirks with it. Have been doing some "light" reading into rollup but not quite sure yet how we could improve this. One quick observation for now: the `rollup-plugin-typescript` dependency/package is no longer maintained, and has been replaced by `@rollup/plugin-typescript`. ([source](https://github.com/rollup/rollup-plugin-typescript))

File: src/components/AppBar/package.json [hunk 1]
```diff
@@ -1,36 +1,35 @@
...
-    "dependencies": {
-        "@gemeente-denhaag/baseprops": "^0.1.7",
-        "@material-ui/core": "^4.11.0"
-    },
-    "peerDependencies": {
-        "react": "^17.0.1"
-    },
-    "devDependencies": {
-        "@gemeente-denhaag/icons": "*",
-        "@gemeente-denhaag/typography": "*"
```
Comment (joostvanviegen) [inline src/components/AppBar/package.json:33]: Wat is de reden dat typography weg is?

File: src/components/Card/tsconfig.json [hunk 2]
```diff
@@ -3,8 +3,8 @@
   "compilerOptions": {
     "rootDir": "src",
     "outDir": "dist",
-    "tsBuildInfoFile": "tsconfig.tsbuildinfo",
-    "composite": true
+    "tsBuildInfoFile": "./dist/tsconfig.tsbuildinfo",
+    "composite": true,
```
Comment (joostvanviegen) [inline src/components/Card/tsconfig.json:7]: Wat is de reden dat composite bij de rest weg is, maar hier niet?

**Review by joostvanviegen — APPROVED** [review]
It looks okay, have made some comments about questions I have regarding some changes.

File: src/components/AppBar/package.json (hunk 1 above)
Comment (bddjong) [reply src/components/AppBar/package.json:33]: It's unused

**Comment by bddjong** [comment]
@GewoonMaarten my IDE complains about the  added svgr rollup plugin not being a function
image

Does this happen for you as well?

**Comment by GewoonMaarten** [comment]
> @GewoonMaarten my IDE complains about the added svgr rollup plugin not being a function
> image
> 
> Does this happen for you as well?

@bddjong no, mine doesn't give any warnings. But it does work, right?

**Comment by bddjong** [comment]
> > @GewoonMaarten my IDE complains about the added svgr rollup plugin not being a function
> > image
> > Does this happen for you as well?
> 
> @bddjong no, mine doesn't give any warnings. But it does work, right?

Yeapp it does

**Comment by GewoonMaarten** [comment]
Sorry for the mess up 😬

**Comment by bddjong** [comment]
Just tested it and ran into no issues (finally). Only thing is that we still have a microsoft copyright notice in some of our bundles. I'm not sure if that's rollup being weird with typescript.

**Review by GewoonMaarten — DISMISSED** [review]

**Review by DanielvanVliet — CHANGES_REQUESTED** [review]
It seems that Proprietary still overwrites index.js, and thus I cant import css from it in a react app.

**Comment by bddjong** [comment]
> It seems that Proprietary still overwrites index.js, and thus I cant import css from it in a react app.

You should be able to import it from the dist for now

**Review by DanielvanVliet — CHANGES_REQUESTED** [review]
The List styling is not loaded entirely, some css rules are missing. Strange behaviour that could be caused by PostCSS.

**Comment by bddjong** [comment]
Tested with verdaccio and confirmed to be working in an external javascript react app. I suggest we merge this and open issues for remaining optimizations so that we can publish the new components.

**Review by GewoonMaarten — APPROVED** [review]

File: src/components/FormControlLabel/src/index.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import MaterialFormControlLabel from '@material-ui/core/FormControlLabel';
 import React, { ReactElement } from 'react';
+import { FormControlLabel as MaterialFormControlLabel } from '@material-ui/core/';
```
Comment (DanielvanVliet) [inline src/components/FormControlLabel/src/index.tsx:2]:
```suggestion
import { FormControlLabel as MaterialFormControlLabel } from '@material-ui/core';
```

**Review by DanielvanVliet — APPROVED** [review]

**Pull request merged**